tests/createGoldenFile.tcl

```diff
@@ -0,0 +1,15 @@
+set_part xc7a35tcpg236-1
```

Comment:
Would be good to stick with `snake_case` instead of `camelCase` to name the files